pixeltable/dataframe.py

```diff
@@ -39,26 +40,33 @@ def _format_img(img: object) -> str:
         img_base64 = base64.b64encode(buffer.getvalue()).decode()
         return f'<img src="data:image/jpeg;base64,{img_base64}">'
 
-def _format_video(video_file_path: str) -> str:
-    # turn absolute video_file_path into relative path, absolute paths don't work
-    p = Path(video_file_path)
-    root = Path(os.getcwd())
-    try:
-        rel_path = p.relative_to(root)
-        return f'<video controls><source src="{rel_path}" type="video/mp4"></video>'
-    except ValueError:
-        # display path as string
-        return video_file_path
+def _get_src(file_path: str) -> str:
```

Comment:
Consider a more specific name (get_src_attr?), 'src' is very general and I had to look at the code to understand what it's returning.